### PR TITLE
fix: consistently display information about 'compared to' and 'synced to' revisions

### LINKED ISSUE #5278: Consistent commit message rendering
**Describe the bug**

The issue is a follow-up for https://github.com/argoproj/argo-cd/pull/5241 . Proposing several additional improvements:

* There are two commit messages on app details page: "compared to" and "synced to" . The popup should be shown for both. 
* Right now popup shows only the first line on the commit message because API truncates it. https://github.com/argoproj/argo-cd/blob/a12f805bb27c443809f32ccd9656d49c60626d6a/reposerver/repository/repository.go#L1259 

Instead, API should return the first message and UI should truncate it on sync panel and show full message in the popup.

* Couple popup panel nitpicks: space is missing between Sync status and dash ( change `Sync OK- Succeeded` to `Sync OK - Succeeded`); Tooltip is no longer required since panel is supposed to show all information:

image

* Refreshing label is not visible anymore ( since panel hight increased a littlebit )
image

### PR DESCRIPTION
**Pull request by alexmt**
Signed-off-by: Alexander Matyushentsev <AMatyushentsev@gmail.com>

Closes https://github.com/argoproj/argo-cd/issues/5278

https://user-images.githubusercontent.com/426437/105795496-cc4c5680-5f41-11eb-9696-4920b97648a9.mov

PR implements changes proposed in https://github.com/argoproj/argo-cd/issues/5278 and additionally replaces popup with sliding panel to be consistent with rest of the UI (e.g. system version info)

### CODE REVIEW

File: ui/src/app/applications/components/application-details/application-details.tsx [hunk 1]
```diff
@@ -435,6 +442,50 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
...
+                                                    <div className='white-box__details'>
+                                                        <div className='row white-box__details-row'>
+                                                            <div className='columns small-3'>Author:</div>
+                                                            <div className='columns small-9'>{this.state.metadata.author}</div>
+                                                        </div>
+                                                    </div>
+                                                    <div className='white-box__details'>
+                                                        <div className='row white-box__details-row'>
+                                                            <div className='columns small-3'>Message:</div>
+                                                            <div className='columns small-9'>
```
Comment (rbreeze) [inline ui/src/app/applications/components/application-details/application-details.tsx:481]: Could we vertically center this? 

Screen Shot 2021-01-26 at 9 06 25 AM

`style={{display: 'flex', align-items: 'center'}}` on the `columns small-9` div would do it

**Review by rbreeze — CHANGES_REQUESTED** [review]
Looks great! Strongly agree with the change to sliding panel. Only one small thing.

File: ui/src/app/applications/components/application-details/application-details.tsx (hunk 1 above)
Comment (alexmt) [reply ui/src/app/applications/components/application-details/application-details.tsx]: great suggestion! Added

**Review by rbreeze — APPROVED** [review]
LGTM!

**Pull request merged**